app/src/main/java/net/gsantner/markor/ui/hleditor/TextActions.java

```diff
@@ -618,6 +620,20 @@ protected boolean runCommonTextAction(final @StringRes int action) {
                 runRenumberOrderedListIfRequired();
                 return true;
             }
+            case R.string.tmaid_common_insert_snippet: {
+                SearchOrCustomTextDialogCreator.showInsertSnippetDialog(_activity, _document.getFormat(), (snip) -> {
+                    _hlEditor.replaceCurrentSelection(StringUtils.interpolateEscapedDateTime(snip));
+                    _appSettings.setLastFormatSnip(_document.getFormat(), snip);
+                });
+                return true;
+            }
+            case R.string.tmaid_common_insert_recent_snippet: {
```

Comment:
I would keep it simple, just one button.

Reply:
this is just used to differentiate the long vs short press.